cherry/src/App.tsx

```diff
@@ -61,7 +61,7 @@ const init: AppState = {
 		{ instruction: { type: "putc_i", i: 110 }, editing: false },
 		{ instruction: { type: "putc_i", i: 100 }, editing: false },
 		{ instruction: { type: "putc_i", i: 33 }, editing: false },
-		{ instruction: {}, editing: true },
+		// { instruction: {}, editing: true },
```

Comment:
this doesn't look docs-related?

Reply:
It's just a bit I had for testing, I meant to remove it before submitting 😅

Reply:
Ahh, if it doesn't break anything, can you take this part of the PR out and land only the doc updates?

Reply:
Is it really that big of a deal?

Reply:
Just trying to keep it fair for everyone

Reply:
If you don't wanna merge it then whatever 😅 I was just here to have fun, I don't really care enough to go back an un-fix something